### DISCUSSION TITLE
Update FLAC to v1.3.4

### DISCUSSION THREAD

**Discussion by bozzis** [enzo1982/freac#330] [Your Ideas]
FLAC version has been updated in February this year to 1.3.4:
![image](https://user-images.githubusercontent.com/24719518/174428117-ee437131-84cd-4151-9fbd-e7e0abf05b71.png)

Currently fre:ac uses version 1.3.3:
![image](https://user-images.githubusercontent.com/24719518/174428134-d1c22628-2dfd-47c3-a5a9-8649d093ab36.png)

Please disregard or mark this post as solved if it's already been done for the next fre:ac update.

**Comment by enzo1982** [comment]
FLAC 1.3.4 is already included in fre:ac's [continuous builds](https://github.com/enzo1982/freac/releases/continuous).
